hazelcast/src/main/java/com/hazelcast/jet/impl/JetServiceBackend.java

```diff
@@ -379,4 +406,58 @@ public void startScanningForJobs() {
         jobCoordinationService.startScanningForJobs();
     }
 
+    // Store the metadata about the job jar that is uploaded from client side
+    public void storeJobMetaData(JobMetaDataParameterObject parameterObject) {
+        checkIfCanExecuteJar();
+        jobUploadStore.processJobMetaData(parameterObject);
+    }
+
+    // Store a part of job jar that is uploaded
```

Comment:
better to change it to JavaDoc with proper `throws` section

Reply:
Ok. Added javadoc